### PR TITLE
rename irstate.d to stmtstate.d

### PR DESCRIPTION
**Pull request by WalterBright**
to match its only declaration

### CODE REVIEW

**Comment by Geod24** [comment]
Can you also remove the boilerplate in the module documentation ? Otherwise LGTM

**Comment by WalterBright** [comment]
Please be specific.

**Comment by Geod24** [comment]
Remove line 2 and 3 in `stmtstate.d`.
https://github.com/dlang/dmd/pull/10984/files#r399867907

**Review by andralex — APPROVED** [review]

**Comment by WalterBright** [comment]
Ack, why wait 3 days?

File: src/dmd/stmtstate.d [hunk 1]
```diff
@@ -1,17 +1,16 @@
 /**
  * Used to help transform statement AST into flow graph.
  * Compiler implementation of the
```
Comment (Geod24) [inline src/dmd/stmtstate.d]: ?

Comment (WalterBright) [reply src/dmd/stmtstate.d]: Need to know where the 'break' statement winds up, for example.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**